labrad/server.py

```diff
@@ -227,6 +227,25 @@ def __init__(self):
         self.signals = []
         self.contexts = {}
 
+    def configure_tls(self, host, tls):
+        """Configure TLS options for this server's connection to labrad.
+
+        Must be called before we actually initiate a network connection, e.g.
+        before this server instance is passed to one of the reactor.connect*
+        methods.
+
+        Args:
+            host (string): The remote hostname of the manager we are onnecting
+                to. This will be used to validate the hostname of the
+                certificate presented by the manager.
+            tls_mode (string): One of 'on', 'off' or 'starttls'. Note that if
+                the tls is 'on', the connection should be initiated with
```

Comment:
not sure how pedantic you want to be - I usually try to stick with using caps rather than no caps for common abbreviations, e.g. "TLS" vs "tls" (in comments not variable names).